scripts/local/launch/helpers/create-azure-resources.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
You need to do a rename.

Reply:
I didn't get this

Reply:
create_azure_resources.sh was deleted and this file shows up as add. But it should be git mv which renames the file and only diffs would show if you made any changes.

Reply:
I deleted and recreated the file with the same name.